tamr_client/project.py

```diff
@@ -12,6 +13,12 @@ class NotFound(TamrClientException):
     pass
 
 
+class Conflict(TamrClientException):
```

Comment:
We have an existing convention of calling this sort of error and `AlreadyExists` error:

```suggestion
class AlreadyExists(TamrClientException):
```

Reply:
changed